src/local.rs

```diff
@@ -44,6 +44,24 @@ pub enum Local {
         #[clap(long)]
         package_image: Option<String>,
     },
+
+    /// Cleanup the resources created by a packaged AppInstance.
+    ///
+    /// This removes all created resource, except the containing Namespace as it
+    /// is created outside of an applyset.
+    Cleanup {
```

Comment:
I think this command should be called `delete`, to remain similar to `kubectl` (apply and delete)

Reply:
Agreed!

I initially called the file `delete.rs` too and was flip-flopping about what it should be called, then I renamed everything to `cleanup` :laughing: 

![image](https://github.com/kubecfg/kubit/assets/56563911/6a05dcc1-03bf-4875-b484-b6ab34581e04)


I should've gone the other way :facepalm: